client/src/styles/theme.js

```diff
@@ -0,0 +1,11 @@
+import { extendTheme } from "@chakra-ui/react";
+
+export const budgetTheme = extendTheme({
```

Comment:
we wanted to use arrow functions to show usage of a newer way to use functions. Would you mind refactoring it?